Helper.cs

```diff
@@ -0,0 +1,64 @@
+﻿
+public class Helper
+{
+    protected List<char> _listOfLettersToCount;
+    protected List<char> _listOfNumberToCount;
+    private List<char> _illegalalCharacters;
+    private int? _result;
+
+    public Helper()
+    {
+        _listOfLettersToCount = ['a', 'b', 'c'];
+        _listOfNumberToCount = ['1', '2', '3'];
+        _illegalalCharacters = ['-', '<', '>'];
+    }
+
+    public bool IsCharacterAndIfINeedToCount(char character)
+    {
+        if (char.IsAsciiLetter(character)) 
+        {
+            var letter = character;
+
+            if (_listOfLettersToCount.Contains(letter))
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        return false;
+    }
+
+    public bool IsDigitAndIfItIsInTheList(char character)
+    {
+        if (char.IsDigit(character))
+        {
+            var letter = character;
+
+            if (_listOfLettersToCount.Contains(letter))
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        return false;
+    }
+
+    public void IncreaseDigitCount(int digitCount, bool b)
+    {
+        if (b)
+        {
+            digitCount++;
+        }
+    }
```

Comment:
digitCount shoud be changed to _resultDigits

Reply:
Why ?

Reply:
because it is internal method variable and increasing it does no differences. If you whant to update count ant have global variable it is better to use it, or use reference type to pass it then it would update the value that is passed

Reply:
What is the difference between value and reference types ?

Reply:
value can be edited in local scope, but edit does not have any effect in main program variable. 
But when using reference all changes to variable has effect in all scope